docs/_static/custom.css

```diff
@@ -0,0 +1,9 @@
+html[data-theme="light"] {
+    --pst-color-primary: #800080;
+    --pst-color-secondary: #ed4c5c;
```

Comment:
:+1: for adjusting the theme colour.

But why did you pick a secondary colour that is so similar to Scitacean?

Reply:
> But why did you pick a secondary colour that is so similar to Scitacean?

No idea, I looked at some palettes online and tried to find a colour that would go well.

Reply:
Could pick some blue/green/turquoise color maybe?